### ISSUE TITLE
Unable to add database or change master password on firstboot

### ISSUE THREAD

**Issue by DocCyblade** [DocCyblade/tkl-odoo#10]
Can not seem to change the database admin password from the admin screen
nor
add new database

From comments:
https://github.com/DocCyblade/tkl-odoo/issues/7#issuecomment-144091097
https://github.com/DocCyblade/tkl-odoo/issues/7#issuecomment-144092221

**Comment by DocCyblade** [comment]
I was able to see this first hand. File permissions on the config file is good. I thought that doing a stop/start after the admin password changed would be enough. I am thinking it's not working all that well. I may try and recycle the service using the init script instead of systemctl command. Not sure if there is a disconnect or not. 

Just as I was typing I doubled checked, I only put the restart in 20-odoo-db-secrets, will also need it in 40-odoo-adminpassword. Pretty certain this will fix this issue.

**Comment by l-arnold** [comment]
Makes sense. 
I still don't understand how the systemctl command is better however.  I think it would work well to use the "/bin/bash /etc/init.d/openerp-server restart" command.

At least need to verify that systemctl restart would take certain variables like init.d/openerp-server  (perhaps they are the same).  Are they the same, or is the service diff than the init.d ?

**Comment by l-arnold** [comment]
note that I could change the admin password but needed to

cd /etc/init.d
/bin/bash/openerp-server restart

After that the Odoo Database Admin commands (new database, change password) were working for me.

Not sure if you are in beta 4?

**Comment by DocCyblade** [comment]
systemd's systemctl command will in some way call the init.d script. We used to in v13 use `service` command. 

The init.d script will pass options to the server script but systemctl does not from what I read. So if you wanted to you could 

```
./etc/init.d/openerp-server stop
./etc/init.d/openerp-server start -u {or other options}
```

**Comment by DocCyblade** [comment]
I'll release v0.4.1 and yes I am working in the dev-beta-4 branch

**Comment by l-arnold** [comment]
so call dev-beta-4 to a new folder/build for tkldev.

Let me know when to try..  It will be a few hours before I can get to it just now.

(in terms of "fixing" some things on my repository, what are the files to focus on trying to get the service configured correctly?  Just a few that I should look at?  I think my issues there are close to the same as those here actually so if we get working here  would like to fix those (but keep the width of what we have there)

**Comment by l-arnold** [comment]
We should add a "checklist" item that we can 
stop start restart
via Both Mechanisms.
Ideally some confirmation (like a running or stopped website) should be used to test.

**Comment by DocCyblade** [comment]
Fixed with v0.4.1
